ext/nnstreamer/tensor_source/tensor_src_grpc.c

```diff
@@ -493,14 +509,23 @@ gst_tensor_src_grpc_set_property (GObject * object, guint prop_id,
       silent_debug ("Set server = %d", self->server);
       break;
     case PROP_HOST:
+    {
+      gchar * host;
+
       if (!g_value_get_string (value)) {
         ml_logw ("host property cannot be NULL");
         break;
       }
-      g_free (self->host);
-      self->host = g_value_dup_string (value);
-      silent_debug ("Set host = %s", self->host);
+      host = g_value_dup_string (value);
```

Comment:
Why value cannot be null?

Reply:
Right, I will remove warning message and regard setting to NULL as the unchanged action.